torch/csrc/distributed/rpc/functions.cpp

```diff
@@ -8,28 +8,56 @@ void processRequestBlocking(
     const std::string& from, Message&& request, RpcAgent& agent) {
   switch (request.type()) {
     case MessageType::SCRIPT_CALL: {
-      ScriptCall op = ScriptCall::fromMessage(request);
+      try {
+        ScriptCall op = ScriptCall::fromMessage(request);
 
-      auto stack = op.stack();
-      op.op()->getOperation()(stack);
-      AT_ASSERT(stack.size() == 1, "Return value of a builtin operator or a "
-          "TorchScript function should be a single IValue, got a vector of "
-          "size ", stack.size());
+        auto stack = op.stack();
+        op.op()->getOperation()(stack);
+        AT_ASSERT(stack.size() == 1, "Return value of a builtin operator or a "
+            "TorchScript function should be a single IValue, got a vector of "
+            "size ", stack.size());
 
-      auto response = ScriptRet(std::move(stack.front())).toMessage();
-      response.setId(request.id());
-      agent.send(from, std::move(response));
+        auto response = ScriptRet(std::move(stack.front())).toMessage();
+        response.setId(request.id());
+        agent.send(from, std::move(response));
+      } catch (std::exception& e) {
+        std::vector<torch::Tensor> tensorTable;
+        std::vector<char> payload;
+        const char* err = e.what();
+        payload.insert(payload.end(), err, err + strlen(err));
+        agent.send(
+            from,
+            Message(
+                std::move(payload),
+                std::move(tensorTable),
```

Comment:
nit: call use `std::vector<torch::Tensor>()` here instead of a separate `tensorTable` var.